google-cloud-bigquery/src/benchmark/java/com/google/cloud/bigquery/benchmark/Benchmark.java

```diff
@@ -66,8 +71,8 @@ public static void main(String[] args) throws Exception {
       }
       Duration total = Duration.between(start, clock.instant());
 
-      double firstByteSec = (double)(firstByte.getNano())/1000/1000/1000 + firstByte.getSeconds();
-      double totalSec = (double)(total.getNano())/1000/1000/1000 + total.getSeconds();
+      double firstByteSec = (double)(firstByte.getNano())/BILLION + firstByte.getSeconds();
+      double totalSec = (double)(total.getNano())/BILLION + total.getSeconds();
```

Comment:
"BILLION" isn't terribly helpful - how about `NS_PER_SECOND` or something like that?